pallets/phala/src/compute/stake_pool_v2.rs

```diff
@@ -543,6 +548,33 @@ pub mod pallet {
 			Ok(())
 		}
 
+		#[pallet::weight(0)]
```

Comment:
Need to make it transactional, because 558 can pass but 559 may fail.

Reply:
done